pkg/controller/migmigration/task.go

```diff
@@ -568,6 +604,11 @@ func (t *Task) UID() string {
 	return string(t.Owner.UID)
 }
 
+// Get whether the migration is cancelled.
+func (t *Task) cancel() bool {
```

Comment:
The method is answering a question instead of performing an action. Recommend `canceled()` would be clearer.